### PR TITLE
feat(payment): INT-6482 | Worldpay - Increase coverage in WorldpayCreditCardPaymentMethod

### PR DESCRIPTION
**Pull request by roberbnd**
## What? [INT-6482](https://bigcommercecloud.atlassian.net/browse/INT-6482)
Test cases added to improve code coverage

## Why?
Code coverage was fulfilling expectations for the strategy.

## Testing / Proof
[Coverage](https://drive.google.com/file/d/1oDyqc9gqL91HP2iRKlA1N8_8-czS9VY0/view?usp=sharing)

@bigcommerce/checkout @bigcommerce/apex-integrations

### CODE REVIEW

**Review by mauricio-sg — APPROVED** [review]
LGTM

File: packages/core/src/app/payment/paymentMethod/WorldpayCreditCardPaymentMethod.tsx [hunk 1]
```diff
@@ -58,8 +58,8 @@ const WorldpayCreditCardPaymentMethod: FunctionComponent<
     }, [initializePayment]);
 
     const appendPaymentPageContent = useCallback(() => {
-        if (threeDSecureVerification) {
-            ref.current.paymentPageContentRef.current?.appendChild(threeDSecureVerification);
+        if (threeDSecureVerification && ref.current.paymentPageContentRef.current) {
+            ref.current.paymentPageContentRef.current.appendChild(threeDSecureVerification);
```
Comment (animesh1987) [inline packages/core/src/app/payment/paymentMethod/WorldpayCreditCardPaymentMethod.tsx:62]: Can I please ask what's the difference in the two statements?

Comment (roberbnd) [reply packages/core/src/app/payment/paymentMethod/WorldpayCreditCardPaymentMethod.tsx:62]: Code coverage:
image

**Comment by roberbnd** [comment]
@animesh1987 could you review please?

**Review by animesh1987 — APPROVED** [review]

**Pull request merged**